src/boot/efi/test-bcd.c

```diff
@@ -152,7 +152,7 @@ TEST(argv_bcds) {
 
                 char16_t *title = get_bcd_title(bcd, len);
                 if (title) {
-                        _cleanup_free_ char *title_utf8 = utf16_to_utf8(title, char16_strlen(title) * 2);
```

Comment:
I'd maybe leave the use of `char16_strlen()` in tests (here and in test-utf8 below) as is, just to make sure `char16_strlen()` does what it should (both the "regular" and EFI variants).

Reply:
it's still in there and being executed and tested, just inside of ut16_to_utf().

Reply:
`utf16_to_utf8()` doesn't seem to use `char16_strlen()` internally. `char16_strlen()` is used only in a couple of places and none of them are in the `utf16_to_utf8()` codepath, unless I'm missing something.

Reply:
Ugh, I read the patch multiple times and somehow missed that, sorry about that :)